src/components/site-form.tsx

```diff
@@ -352,6 +395,50 @@ export const SiteForm = ( {
 											value={ sitePath }
 											onClick={ onSelectPath }
 										/>
+										{ wpVersionsEnabled && allowVersionsChange && (
```

Comment:
I don't fully understand why we need the `allowVersionsChange` prop.

Reply:
We can also remove that once the edit page component has the versions.
I added more details in TODO comments.